components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -75,11 +87,11 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
     }
 
     isCollapsed = (props: Props) => {
-        return props.isCategoryCollapsed && !this.isUnread() && !props.isCurrentChannel;
+        return /*props.isDropDisabled || */props.isCategoryDragged || (props.isCategoryCollapsed && !this.isUnread() && !props.isCurrentChannel);
```

Comment:
Remember to remove the commented out code once we've got the correct behaviour here.